### ISSUE TITLE
MQTT setzt aus

### ISSUE THREAD

**Issue by ESP2000** [jomjol/AI-on-the-edge-device#373]
Hallo...,

nutze das System für die Erfassung Wasserverbrauch, finde die Lösung toll. Habe das Problem, dass die MQTT Übertragung immer wieder für längere Zeit 2-4 Stunden unterbrochen ist. WLAN Verbindung ist stabil, der Zugriff auf das WEB-Interface ist auch möglich, wenn keine MQTT Daten gesendet werden.
Verwende die Version 8.5.0, Intervall. 3.0 min

**Comment by zumheulen** [comment]
Hello, I also have problems with MQTT.
I suspected it was due to Homeassistant, but the MQTT Explorer also often pulses, but irregularly no new MQTT data.
I look at the log file, but I interpret what he sends...
Here is an excerpt from my log.

> 2022-10-07T13:04:31: task_autodoFlow - round done
2022-10-07T13:04:31: CPU Temperature: 30.0
2022-10-07T13:07:38: 	handler_wasserzaehler - Start
2022-10-07T13:07:38: 	handler_wasserzaehler - Start
2022-10-07T13:07:38: 	handler_wasserzaehler - Start
2022-10-07T13:07:38: 	handler_cputemp - Start
2022-10-07T13:07:38: 	handler_cputemp - End
2022-10-07T13:07:38: 	handler_rssi - Start
2022-10-07T13:07:38: 	handler_rssi - End
2022-10-07T13:07:38: 	handler_wasserzaehler - Start
2022-10-07T13:07:39: 	handler_prevalue - Start
2022-10-07T13:07:39: 	handler_prevalue - Start
2022-10-07T13:08:56: task_autodoFlow - next round - Round #79
2022-10-07T13:08:56: 	FlowControll.doFlow - ClassFlowMakeImage
2022-10-07T13:09:04: 	FlowControll.doFlow - ClassFlowAlignment
2022-10-07T13:09:25: 	FlowControll.doFlow - ClassFlowCNNGeneral
2022-10-07T13:09:25: ClassFlowCNNGeneral::doFlow nach Alignment
2022-10-07T13:09:27: 	FlowControll.doFlow - ClassFlowCNNGeneral
2022-10-07T13:09:27: ClassFlowCNNGeneral::doFlow nach Alignment
2022-10-07T13:09:28: Nach Invoke
2022-10-07T13:09:29: Nach Invoke
2022-10-07T13:09:29: Nach Invoke
2022-10-07T13:09:30: Nach Invoke
2022-10-07T13:09:30: 	FlowControll.doFlow - ClassFlowPostProcessing
2022-10-07T13:09:30: ClassFlowCNNGeneral::getReadout _analog=0, _extendedResolution=0, prev=-1
2022-10-07T13:09:30: ClassFlowCNNGeneral::ZeigerEvalAnalogNeu - kein Vorgänger - Ergebnis = 9 zahl: 9.262082 ziffer_vorgaenger = -1 AnalogFehler = 3.000000
2022-10-07T13:09:30: ClassFlowCNNGeneral::ZeigerEvalAnalogNeu - Zahl uneindeutig, Korrektur nach unten - Ergebnis = 1 zahl: 1.762082 ziffer_vorgaenger = 9 AnalogFehler = 3.000000
2022-10-07T13:09:30: ClassFlowCNNGeneral::ZeigerEvalAnalogNeu - Zahl eindeutig, keine Korrektur notwendig - Ergebnis = 9 zahl: 9.642361 ziffer_vorgaenger = 1 AnalogFehler = 3.000000
2022-10-07T13:09:30: ClassFlowCNNGeneral::ZeigerEvalAnalogNeu - Zahl uneindeutig, Korrektur nach unten - Ergebnis = 2 zahl: 2.885269 ziffer_vorgaenger = 9 AnalogFehler = 3.000000
2022-10-07T13:09:30: ClassFlowCNNGeneral::getReadout _analog=0, _extendedResolution=0, prev=2
2022-10-07T13:09:30: 	FlowControll.doFlow - ClassFlowMQTT
2022-10-07T13:09:30: MQTT - sent publish successful in MQTTPublish, msg_id=37155, homeassistant/zaehler/wasserzaehler/connection, connected
2022-10-07T13:09:30: MQTT - sent publish successful in MQTTPublish, msg_id=27965, homeassistant/zaehler/wasserzaehler/connection, connected
2022-10-07T13:09:30: MQTT - sent publish successful in MQTTPublish, msg_id=38269, homeassistant/zaehler/wasserzaehler/uptime, 23439
2022-10-07T13:09:30: MQTT - sent publish successful in MQTTPublish, msg_id=53267, homeassistant/zaehler/wasserzaehler/freeMem, 3074343
2022-10-07T13:09:30: MQTT - sent publish successful in MQTTPublish, msg_id=45541, homeassistant/zaehler/wasserzaehler/wifiRSSI, -63
2022-10-07T13:09:30: MQTT - sent publish successful in MQTTPublish, msg_id=49682, homeassistant/zaehler/wasserzaehler/CPUtemp, 29.444445
2022-10-07T13:09:30: MQTT - sent publish successful in MQTTPublish, msg_id=52788, homeassistant/zaehler/wasserzaehler/main/error, Neg. Rate - Read:  - Raw: 00111.2919 - Pre: 111.3299 
2022-10-07T13:09:30: MQTT - sent publish successful in MQTTPublish, msg_id=42570, homeassistant/zaehler/wasserzaehler/main/changeabsolut, 0.0008
2022-10-07T13:09:30: MQTT - sent publish successful in MQTTPublish, msg_id=41524, homeassistant/zaehler/wasserzaehler/main/raw, 00111.2919
2022-10-07T13:09:30: MQTT - sent publish successful in MQTTPublish, msg_id=53859, homeassistant/zaehler/wasserzaehler/main/timestamp, 2022-10-07T12:34:04
2022-10-07T13:09:30: MQTT - sent publish successful in MQTTPublish, msg_id=36864, homeassistant/zaehler/wasserzaehler/main/json, {"value":"","raw":"00111.2919","error":"Neg. Rate - Read:  - Raw: 00111.2919 - Pre: 111.3299 ","rate":"","timestamp":"2022-10-07T12:34:04"}
2022-10-07T13:09:31: task_autodoFlow - round done
2022-10-07T13:09:31: CPU Temperature: 29.4
2022-10-07T13:13:38: 	handler_wasserzaehler - Start
2022-10-07T13:13:38: 	handler_wasserzaehler - Start
2022-10-07T13:13:38: 	handler_cputemp - Start
2022-10-07T13:13:38: 	handler_cputemp - End
2022-10-07T13:13:38: 	handler_wasserzaehler - Start
2022-10-07T13:13:38: 	handler_rssi - Start
2022-10-07T13:13:38: 	handler_rssi - End
2022-10-07T13:13:38: 	handler_wasserzaehler - Start
2022-10-07T13:13:38: 	handler_prevalue - Start
2022-10-07T13:13:38: 	handler_prevalue - Start
2022-10-07T13:13:56: task_autodoFlow - next round - Round #80
2022-10-07T13:13:56: 	FlowControll.doFlow - ClassFlowMakeImage
2022-10-07T13:14:04: 	FlowControll.doFlow - ClassFlowAlignment
2022-10-07T13:14:25: 	FlowControll.doFlow - ClassFlowCNNGeneral
2022-10-07T13:14:25: ClassFlowCNNGeneral::doFlow nach Alignment
2022-10-07T13:14:27: 	FlowControll.doFlow - ClassFlowCNNGeneral
2022-10-07T13:14:27: ClassFlowCNNGeneral::doFlow nach Alignment
2022-10-07T13:14:28: Nach Invoke
2022-10-07T13:14:29: Nach Invoke
2022-10-07T13:14:29: Nach Invoke
2022-10-07T13:14:30: Nach Invoke
2022-10-07T13:14:30: 	FlowControll.doFlow - ClassFlowPostProcessing
2022-10-07T13:14:30: ClassFlowCNNGeneral::getReadout _analog=0, _extendedResolution=0, prev=-1
2022-10-07T13:14:30: ClassFlowCNNGeneral::ZeigerEvalAnalogNeu - kein Vorgänger - Ergebnis = 2 zahl: 2.811133 ziffer_vorgaenger = -1 AnalogFehler = 3.000000
2022-10-07T13:14:30: ClassFlowCNNGeneral::ZeigerEvalAnalogNeu - Zahl uneindeutig, Korrektur nach oben - Ergebnis = 3 zahl: 3.055086 ziffer_vorgaenger = 2 AnalogFehler = 3.000000
2022-10-07T13:14:30: ClassFlowCNNGeneral::ZeigerEvalAnalogNeu - Zahl eindeutig, keine Korrektur notwendig - Ergebnis = 9 zahl: 9.688867 ziffer_vorgaenger = 3 AnalogFehler = 3.000000
2022-10-07T13:14:30: ClassFlowCNNGeneral::ZeigerEvalAnalogNeu - Zahl uneindeutig, Korrektur nach unten - Ergebnis = 2 zahl: 2.874160 ziffer_vorgaenger = 9 AnalogFehler = 3.000000
2022-10-07T13:14:30: ClassFlowCNNGeneral::getReadout _analog=0, _extendedResolution=0, prev=2
2022-10-07T13:14:30: 	FlowControll.doFlow - ClassFlowMQTT
2022-10-07T13:14:30: MQTT - sent publish successful in MQTTPublish, msg_id=17950, homeassistant/zaehler/wasserzaehler/connection, connected
2022-10-07T13:14:30: MQTT - sent publish successful in MQTTPublish, msg_id=61820, homeassistant/zaehler/wasserzaehler/connection, connected
2022-10-07T13:14:30: MQTT - sent publish successful in MQTTPublish, msg_id=27089, homeassistant/zaehler/wasserzaehler/uptime, 23739
2022-10-07T13:14:30: MQTT - sent publish successful in MQTTPublish, msg_id=4288, homeassistant/zaehler/wasserzaehler/freeMem, 3073947
2022-10-07T13:14:30: MQTT - sent publish successful in MQTTPublish, msg_id=21993, homeassistant/zaehler/wasserzaehler/wifiRSSI, -62
2022-10-07T13:14:30: MQTT - sent publish successful in MQTTPublish, msg_id=10139, homeassistant/zaehler/wasserzaehler/CPUtemp, 30.000000
2022-10-07T13:14:30: MQTT - sent publish successful in MQTTPublish, msg_id=28310, homeassistant/zaehler/wasserzaehler/main/error, Neg. Rate - Read:  - Raw: 00111.2932 - Pre: 111.3299 
2022-10-07T13:14:30: MQTT - sent publish successful in MQTTPublish, msg_id=43921, homeassistant/zaehler/wasserzaehler/main/changeabsolut, 0.0008
2022-10-07T13:14:30: MQTT - sent publish successful in MQTTPublish, msg_id=20594, homeassistant/zaehler/wasserzaehler/main/raw, 00111.2932
2022-10-07T13:14:30: MQTT - sent publish successful in MQTTPublish, msg_id=40703, homeassistant/zaehler/wasserzaehler/main/timestamp, 2022-10-07T12:34:04
2022-10-07T13:14:30: MQTT - sent publish successful in MQTTPublish, msg_id=12337, homeassistant/zaehler/wasserzaehler/main/json, {"value":"","raw":"00111.2932","error":"Neg. Rate - Read:  - Raw: 00111.2932 - Pre: 111.3299 ","rate":"","timestamp":"2022-10-07T12:34:04"}
2022-10-07T13:14:30: task_autodoFlow - round done
2022-10-07T13:14:30: CPU Temperature: 30.0
2022-10-07T13:18:38: 	handler_wasserzaehler - Start
2022-10-07T13:18:38: 	handler_wasserzaehler - Start
2022-10-07T13:18:38: 	handler_wasserzaehler - Start
2022-10-07T13:18:38: 	handler_rssi - Start
2022-10-07T13:18:38: 	handler_rssi - End
2022-10-07T13:18:38: 	handler_wasserzaehler - Start
2022-10-07T13:18:39: 	handler_prevalue - Start
2022-10-07T13:18:39: 	handler_prevalue - Start
2022-10-07T13:18:39: 	handler_cputemp - Start
2022-10-07T13:18:39: 	handler_cputemp - End
2022-10-07T13:18:56: task_autodoFlow - next round - Round #81
2022-10-07T13:18:56: 	FlowControll.doFlow - ClassFlowMakeImage
2022-10-07T13:19:04: 	FlowControll.doFlow - ClassFlowAlignment
2022-10-07T13:19:25: 	FlowControll.doFlow - ClassFlowCNNGeneral
2022-10-07T13:19:25: ClassFlowCNNGeneral::doFlow nach Alignment
2022-10-07T13:19:27: 	FlowControll.doFlow - ClassFlowCNNGeneral
2022-10-07T13:19:27: ClassFlowCNNGeneral::doFlow nach Alignment
2022-10-07T13:19:28: Nach Invoke
2022-10-07T13:19:29: Nach Invoke
2022-10-07T13:19:30: Nach Invoke
2022-10-07T13:19:30: Nach Invoke
2022-10-07T13:19:30: 	FlowControll.doFlow - ClassFlowPostProcessing
2022-10-07T13:19:30: ClassFlowCNNGeneral::getReadout _analog=0, _extendedResolution=0, prev=-1
2022-10-07T13:19:30: ClassFlowCNNGeneral::ZeigerEvalAnalogNeu - kein Vorgänger - Ergebnis = 2 zahl: 2.294982 ziffer_vorgaenger = -1 AnalogFehler = 3.000000
2022-10-07T13:19:30: ClassFlowCNNGeneral::ZeigerEvalAnalogNeu - Zahl eindeutig, keine Korrektur notwendig - Ergebnis = 5 zahl: 5.427118 ziffer_vorgaenger = 2 AnalogFehler = 3.000000
2022-10-07T13:19:30: ClassFlowCNNGeneral::ZeigerEvalAnalogNeu - Zahl eindeutig, keine Korrektur notwendig - Ergebnis = 9 zahl: 9.693307 ziffer_vorgaenger = 5 AnalogFehler = 3.000000
2022-10-07T13:19:30: ClassFlowCNNGeneral::ZeigerEvalAnalogNeu - Zahl uneindeutig, Korrektur nach unten - Ergebnis = 2 zahl: 2.887569 ziffer_vorgaenger = 9 AnalogFehler = 3.000000
2022-10-07T13:19:30: ClassFlowCNNGeneral::getReadout _analog=0, _extendedResolution=0, prev=2
2022-10-07T13:19:30: 	FlowControll.doFlow - ClassFlowMQTT
2022-10-07T13:19:30: MQTT - sent publish successful in MQTTPublish, msg_id=12900, homeassistant/zaehler/wasserzaehler/connection, connected
2022-10-07T13:19:30: MQTT - sent publish successful in MQTTPublish, msg_id=58719, homeassistant/zaehler/wasserzaehler/connection, connected
2022-10-07T13:19:30: MQTT - sent publish successful in MQTTPublish, msg_id=14167, homeassistant/zaehler/wasserzaehler/uptime, 24039
2022-10-07T13:19:30: MQTT - sent publish successful in MQTTPublish, msg_id=38427, homeassistant/zaehler/wasserzaehler/freeMem, 3075619
2022-10-07T13:19:31: MQTT - sent publish successful in MQTTPublish, msg_id=22086, homeassistant/zaehler/wasserzaehler/wifiRSSI, -65
2022-10-07T13:19:31: MQTT - sent publish successful in MQTTPublish, msg_id=63835, homeassistant/zaehler/wasserzaehler/CPUtemp, 29.444445
2022-10-07T13:19:31: MQTT - sent publish successful in MQTTPublish, msg_id=32674, homeassistant/zaehler/wasserzaehler/main/error, Neg. Rate - Read:  - Raw: 00111.2952 - Pre: 111.3299 
2022-10-07T13:19:31: MQTT - sent publish successful in MQTTPublish, msg_id=55992, homeassistant/zaehler/wasserzaehler/main/changeabsolut, 0.0008
2022-10-07T13:19:31: MQTT - sent publish successful in MQTTPublish, msg_id=61444, homeassistant/zaehler/wasserzaehler/main/raw, 00111.2952
2022-10-07T13:19:31: MQTT - sent publish successful in MQTTPublish, msg_id=30095, homeassistant/zaehler/wasserzaehler/main/timestamp, 2022-10-07T12:34:04
2022-10-07T13:19:31: MQTT - sent publish successful in MQTTPublish, msg_id=3809, homeassistant/zaehler/wasserzaehler/main/json, {"value":"","raw":"00111.2952","error":"Neg. Rate - Read:  - Raw: 00111.2952 - Pre: 111.3299 ","rate":"","timestamp":"2022-10-07T12:34:04"}
2022-10-07T13:19:31: task_autodoFlow - round done
2022-10-07T13:19:31: CPU Temperature: 24.4
2022-10-07T13:23:56: task_autodoFlow - next round - Round #82
2022-10-07T13:23:56: 	FlowControll.doFlow - ClassFlowMakeImage
2022-10-07T13:24:04: 	FlowControll.doFlow - ClassFlowAlignment
2022-10-07T13:24:08: 	handler_wasserzaehler - Start
2022-10-07T13:24:08: 	handler_wasserzaehler - Start
2022-10-07T13:24:08: 	handler_cputemp - Start
2022-10-07T13:24:08: 	handler_cputemp - End
2022-10-07T13:24:08: 	handler_wasserzaehler - Start
2022-10-07T13:24:08: 	handler_rssi - Start
2022-10-07T13:24:08: 	handler_rssi - End
2022-10-07T13:24:08: 	handler_wasserzaehler - Start
2022-10-07T13:24:08: 	handler_prevalue - Start
2022-10-07T13:24:08: 	handler_prevalue - Start
2022-10-07T13:24:25: 	FlowControll.doFlow - ClassFlowCNNGeneral
2022-10-07T13:24:25: ClassFlowCNNGeneral::doFlow nach Alignment
2022-10-07T13:24:27: 	FlowControll.doFlow - ClassFlowCNNGeneral
2022-10-07T13:24:27: ClassFlowCNNGeneral::doFlow nach Alignment
2022-10-07T13:24:28: Nach Invoke
2022-10-07T13:24:29: Nach Invoke
2022-10-07T13:24:30: Nach Invoke
2022-10-07T13:24:30: Nach Invoke
2022-10-07T13:24:30: 	FlowControll.doFlow - ClassFlowPostProcessing
2022-10-07T13:24:30: ClassFlowCNNGeneral::getReadout _analog=0, _extendedResolution=0, prev=-1
2022-10-07T13:24:31: ClassFlowCNNGeneral::ZeigerEvalAnalogNeu - kein Vorgänger - Ergebnis = 2 zahl: 2.285781 ziffer_vorgaenger = -1 AnalogFehler = 3.000000
2022-10-07T13:24:31: ClassFlowCNNGeneral::ZeigerEvalAnalogNeu - Zahl eindeutig, keine Korrektur notwendig - Ergebnis = 5 zahl: 5.389895 ziffer_vorgaenger = 2 AnalogFehler = 3.000000
2022-10-07T13:24:31: ClassFlowCNNGeneral::ZeigerEvalAnalogNeu - Zahl eindeutig, keine Korrektur notwendig - Ergebnis = 9 zahl: 9.696198 ziffer_vorgaenger = 5 AnalogFehler = 3.000000
2022-10-07T13:24:31: ClassFlowCNNGeneral::ZeigerEvalAnalogNeu - Zahl uneindeutig, Korrektur nach unten - Ergebnis = 2 zahl: 2.885269 ziffer_vorgaenger = 9 AnalogFehler = 3.000000
2022-10-07T13:24:31: ClassFlowCNNGeneral::getReadout _analog=0, _extendedResolution=0, prev=2
2022-10-07T13:24:31: 	FlowControll.doFlow - ClassFlowMQTT
2022-10-07T13:24:31: MQTT - sent publish successful in MQTTPublish, msg_id=24324, homeassistant/zaehler/wasserzaehler/connection, connected
2022-10-07T13:24:31: MQTT - sent publish successful in MQTTPublish, msg_id=7031, homeassistant/zaehler/wasserzaehler/connection, connected
2022-10-07T13:24:31: MQTT - sent publish successful in MQTTPublish, msg_id=14472, homeassistant/zaehler/wasserzaehler/uptime, 24340
2022-10-07T13:24:31: MQTT - sent publish successful in MQTTPublish, msg_id=13563, homeassistant/zaehler/wasserzaehler/freeMem, 3075627
2022-10-07T13:24:31: MQTT - sent publish successful in MQTTPublish, msg_id=43952, homeassistant/zaehler/wasserzaehler/wifiRSSI, -64
2022-10-07T13:24:31: MQTT - sent publish successful in MQTTPublish, msg_id=28744, homeassistant/zaehler/wasserzaehler/CPUtemp, 29.444445
2022-10-07T13:24:31: MQTT - sent publish successful in MQTTPublish, msg_id=26667, homeassistant/zaehler/wasserzaehler/main/error, Neg. Rate - Read:  - Raw: 00111.2952 - Pre: 111.3299 
2022-10-07T13:24:31: MQTT - sent publish successful in MQTTPublish, msg_id=11987, homeassistant/zaehler/wasserzaehler/main/changeabsolut, 0.0008
2022-10-07T13:24:31: MQTT - sent publish successful in MQTTPublish, msg_id=51942, homeassistant/zaehler/wasserzaehler/main/raw, 00111.2952
2022-10-07T13:24:31: MQTT - sent publish successful in MQTTPublish, msg_id=50769, homeassistant/zaehler/wasserzaehler/main/timestamp, 2022-10-07T12:34:04
2022-10-07T13:24:31: MQTT - sent publish successful in MQTTPublish, msg_id=63651, homeassistant/zaehler/wasserzaehler/main/json, {"value":"","raw":"00111.2952","error":"Neg. Rate - Read:  - Raw: 00111.2952 - Pre: 111.3299 ","rate":"","timestamp":"2022-10-07T12:34:04"}
2022-10-07T13:24:31: task_autodoFlow - round done
2022-10-07T13:24:31: CPU Temperature: 30.0
2022-10-07T13:25:57: info_get_handler
2022-10-07T13:25:57: info_get_handler
2022-10-07T13:25:57: info_get_handler
2022-10-07T13:25:58: info_get_handler
2022-10-07T13:25:58: info_get_handler
2022-10-07T13:25:58: info_get_handler
2022-10-07T13:25:58: info_get_handler
2022-10-07T13:25:58: info_get_handler
2022-10-07T13:26:02: download_get_handler
2022-10-07T13:26:10: download_get_handler
2022-10-07T13:26:38: download_get_handler
2022-10-07T13:26:40: download_get_handler
